email_routing_destination.go

```diff
@@ -0,0 +1,146 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type EmailRoutingDestinationAddress struct {
+	Tag      string     `json:"tag,omitempty"`
+	Email    string     `json:"email,omitempty"`
+	Verified *time.Time `json:"verified,omitempty"`
+	Created  *time.Time `json:"created,omitempty"`
+	Modified *time.Time `json:"modified,omitempty"`
+}
+
+type EmailRoutingListAddressParameters struct {
+	ResultInfo
+}
+
+type EmailRoutingListAddressResponse struct {
+	Result     []EmailRoutingDestinationAddress `json:"result"`
+	ResultInfo `json:"result_info"`
+	Response
+}
+
+type EmailRoutingCreateAddressParameters struct {
+	Email string `json:"email,omitempty"`
+}
+
+type EmailRoutingCreateAddressResponse struct {
+	Result EmailRoutingDestinationAddress `json:"result,omitempty"`
+	Response
+}
+
+// EmailRoutingListDestinationAddresses Lists existing destination addresses.
+//
+// API reference: https://api.cloudflare.com/#email-routing---destination-addresses-list-destination-addresses
+func (api *API) EmailRoutingListDestinationAddresses(ctx context.Context, rc *ResourceContainer, params EmailRoutingListAddressParameters) ([]EmailRoutingDestinationAddress, *ResultInfo, error) {
```

Comment:
can we make these method names `<verb><thing>` so `ListEmailRoutingDestinationAddresses`? it reads and makes the intent a little clearer.